index.html

```diff
@@ -25,12 +27,185 @@ <h2 class="padding">I’m Youssef Glad to see you!</h2>
       </p>
       <a class="padding" href="#">LET’S CONNECT</a>
       <div class="social">
-        <img src="/assets/127.svg" alt="" />
-        <img class="sc-item" src="/assets/125.svg" alt="" />
-        <img class="sc-item" src="/assets/124.svg" alt="" />
-        <img class="sc-item" src="/assets/126.svg" alt="" />
-        <img class="sc-item" src="/assets/123.svg" alt="" />
+        <img src="./assets/127.svg" alt="" />
+        <img class="sc-item" src="./assets/125.svg" alt="" />
+        <img class="sc-item" src="./assets/124.svg" alt="" />
+        <img class="sc-item" src="./assets/126.svg" alt="" />
+        <img class="sc-item" src="./assets/123.svg" alt="" />
       </div>
     </section>
+    <!-- Works Section -->
+    <section class="works">
+      <div class="cards">
+        <!-- First Card -->
+        <div class="card">
+          <img class="card-img" src="./assets/Snapshoot1.png" alt="" />
+          <div class="card-details">
+            <div class="card-item-mg">
+              <span class="card-title">Tonic</span>
+              <div class="card-info">
+                <span class="info-active">CANOPY</span>
+                <img src="./assets/Counter.svg" alt="" />
+                <span>Back End Dev</span>
+                <img src="./assets/Counter.svg" alt="" />
+                <span>2015</span>
+              </div>
+            </div>
+            <p class="card-descrip card-item-mg">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="card-tags card-item-mg">
+              <span class="card-tag">html</span>
+              <span class="card-tag">css</span>
+              <span class="card-tag">javascript</span>
+            </div>
```

Comment:
- [ ] Please make sure you use the `ul` and `li` tags to list the tagged languages in your project. You can read more about it [here](https://www.w3schools.com/html/html_lists.asp) :pray:

Reply:
fixed